dz_dynamodb/sync_strategies/log_based.py

```diff
@@ -147,6 +156,13 @@ def sync(config, state, stream):
     table_name = stream['tap_stream_id']
 
     client = dynamodb.get_client(config)
+    # for open shards their will be a continous loop while getting records so to handle that max_iteration_for_open_shards is used
+    found_max_iteration_for_open_shards = False
+    max_iteration_for_open_shards = 1000
```

Comment:
why have we chosen 1000? any specific doc or significance for it?

Reply:
it is by default we are marking client can change from configuration